client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/index.tsx

```diff
@@ -44,7 +62,7 @@ const SiteVertical: Step = function SiteVertical( { navigation } ) {
 			setIsBusy( true );
 			await saveSiteSettings( site.ID, {
 				site_vertical_id: value,
-				blogname: label,
+				blogname: siteTitleText,
```

Comment:
It might be weird to update the blog name here as the goals/intent is irrelevant to this step. It might be better to update this when the user finishes the goals/intent step. What do you think?

Reply:
You're right, this logic makes more sense in the goals step 🙂